server/scsynth/SC_UnitDef.cpp

```diff
@@ -106,6 +107,7 @@ bool PlugIn_DefineCmd(const char* inCmdName, PlugInCmdFunc inFunc, void* inUserD
 int Unit_DoCmd(World* inWorld, int inSize, char* inData) {
     sc_msg_iter msg(inSize, inData);
     int nodeID = msg.geti();
+    gMissingNodeID = nodeID;
```

Comment:
I don't see it ever being used – where does `gMissingNodeID` reappear?

Reply:
https://github.com/supercollider/supercollider/blob/608bb981162c2c26f0a32c09d82557b29774a32e/common/SC_Errors.cpp#L26

and

https://github.com/supercollider/supercollider/blob/608bb981162c2c26f0a32c09d82557b29774a32e/server/scsynth/SC_MiscCmds.cpp#L48